website/pages/docs/plugins/sources/googleanalytics/overview.mdx

```diff
@@ -0,0 +1,27 @@
+---
+name: Google Analytics
+---
+# Google Analytics Source Plugin
+
+import { getLatestVersion } from "../../../../../utils/versions";
+import { Badge } from "../../../../../components/Badge";
+import Configuration from "./_configuration.mdx";
+
+<Badge text={"Latest: " + getLatestVersion("source", `googleanalytics`)}/>
+
+The CloudQuery Google Analytics plugin allows you to run custom
+[Google Analytics Data API v1 reports](https://developers.google.com/analytics/devguides/reporting/data/v1/basics#reports)
+from [Google Analytics](https://www.google.com/analytics) and load it into any supported CloudQuery destination (e.g. PostgreSQL, BigQuery, Snowflake, and [more](/docs/plugins/destinations/overview)).
+
+## Example
+
+<Configuration/>
+
+## Configuration
+
+See [Configuration](/docs/plugins/sources/googleanalytics/configuration).
+
+## Authentication
+
+See [Authentication](/docs/plugins/sources/googleanalytics/authentication).
+
```

Comment:
```suggestion
```

I think we can remove this (we don't do it in other plugins). The links are already in the side bar and in the pagination section:
![image](https://user-images.githubusercontent.com/26760571/224013106-eabea4b0-0b09-49f1-8f9a-9c5a4a55f6c9.png)